static/src/js/schemas/uiSchemas/services/service_options.js

```diff
@@ -0,0 +1,286 @@
+const serviceOptionsUiSchema = {
```

Comment:
The sidebar has this labeled as just 'Options'. Should we change the side bar or this form heading for consistency?